### ISSUE TITLE
Drivelist crash with PocketBook device

### ISSUE THREAD

**Issue by johanpoirier** [resin-io-modules/drivelist#261]
Drivelist (v6.1.1 on MacOS 10.13.3) crashes if I connect my e-ink device (a PocketBook Touch Lux 2).

```
$ node test.js 
/Users/dummy/projects/app/node_modules/drivelist/lib/diskutil.js:122
    const partitions = layout.Partitions.concat(layout.APFSVolumes || []);
                                         ^

TypeError: Cannot read property 'concat' of undefined
    at devices.forEach (/Users/dummy/projects/app/node_modules/drivelist/lib/diskutil.js:122:42)
    at Array.forEach (<anonymous>)
    at setMountpoints (/Users/dummy/projects/app/node_modules/drivelist/lib/diskutil.js:118:11)
    at asyncMap (/Users/dummy/projects/app/node_modules/drivelist/lib/diskutil.js:149:7)
    at next (/Users/dummy/projects/app/node_modules/drivelist/lib/diskutil.js:73:7)
    at ChildProcess.childProcess.spawn.on.on (/Users/dummy/projects/app/node_modules/drivelist/lib/diskutil.js:45:7)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
```

Content of `test.js`:
```javascript
const drivelist = require('drivelist');

drivelist.list((error, drives) => {
  if (error) {
    throw error;
  }

  console.log(drives);
});
```

Thanks for your help.

**Comment by johanpoirier** [comment]
I just tried the same script with the 5.2.12 version: it does not crash this time.
But the mountpoint path is incomplete.

The real volume path:
```
$ ls -l /Volumes/
total 16
lrwxr-xr-x  1 root          wheel     1 Mar  1 07:36 Macintosh HD -> /
drwxrwxrwx@ 1 dummy  staff  4096 Mar 16 10:31 TL TEA
```

The one found by drivelist:
```
$ node test.js 
[]
[ { path: '/' }, { path: '/private/var/vm' } ]
[ { path: '/Volumes/TL' } ]
```

With test.js:
```javascript
const drivelist = require('drivelist');

drivelist.list((error, drives) => {
  if (error) {
    throw error;
  }

  drives.forEach(drive => console.dir(drive.mountpoints));
});
```

**Comment by lurch** [comment]
The problem with spaces in the mountpoint was fixed in #246 which got released in drivelist 6.0.1 :slightly_smiling_face:

**Comment by johanpoirier** [comment]
> The problem with spaces in the mountpoint was fixed in #246 which got released in drivelist 6.0.1

OK thanks. That's why I wanted to use the 6.x version but with no luck.

**Comment by lurch** [comment]
Maybe this issue is a duplicate of #258 ?

**Comment by johanpoirier** [comment]
> Maybe this issue is a duplicate of #258 ?

Yes it looks like it. Can I do something to help?

**Comment by jhermsmeier** [comment]
Duplicate of #258

**Comment by jhermsmeier** [comment]
Being fixed in #259
